ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-112 Add Interest of Justice page

## Description of change
Adds ioj model and table - associated to case
Adds ioj controller and view

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-112
Designs: https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?node-id=2711%3A6495

## Screenshots of changes (if applicable)
<img width="986" alt="image" src="https://user-images.githubusercontent.com/45827968/196996748-16321921-e5c1-49c9-ad50-9f1b38d9d94e.png">

## How to manually test the feature
Go through full flow
Check validation error by submitting without selecting an ioj
Reload page, check validation error for submitting with selected ioj but no description
Reload page, select ioj, enter text in text area, submit, check record is correct in console
